spec/models/course_spec.rb

```diff
@@ -194,4 +194,10 @@
       end
     end
   end
+
+  describe "#provider_type" do
+    it "returns the provider type" do
+      expect(course.provider_type).to eq("providers")
```

Comment:
why is that `providers`?

Reply:
Ah good spot. This Course model and spec has been corrected